configs/ruby/GPU_VIPER.py

```diff
@@ -241,7 +242,8 @@ class SQCCache(RubyCache):
     def create(self, options):
         self.size = MemorySize(options.sqc_size)
         self.assoc = options.sqc_assoc
-        self.replacement_policy = RP_choose(options.tcp_rp)
+        if hasattr(options, "tcp_rp"):
```

Comment:
I think this needs to be sqc_rp?

Reply:
Yea, I think this was incorrect before

Reply:
Just to make sure I'm not going crazy: did you push a change to update this?  Or just add a comment?  If the former, I do not see it ...

Reply:
Still working on it, should be up now